Sources/Recycling/ListViews/Basic/BasicTableView.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public protocol BasicTableViewDelegate: AnyObject {
+    func basicTableView(_ basicTableView: BasicTableView, didSelectItemAtIndex index: Int)
+}
+
+open class BasicTableViewItem: BasicTableViewCellViewModel {
+    open var title: String
+    open var subtitle: String?
+    open var detailText: String?
+    open var hasChevron: Bool
+    open var isEnabled: Bool = true
+
+    public init(title: String) {
+        self.title = title
+        self.hasChevron = false
+    }
+}
+
+open class BasicTableView: ShadowScrollView {
+    public static let estimatedRowHeight: CGFloat = 60.0
+    open var selectedIndexPath: IndexPath?
+    open var items: [BasicTableViewItem]
+
+    // MARK: - Internal properties
+
+    private lazy var tableView: UITableView = {
+        let tableView = UITableView(withAutoLayout: true)
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.backgroundColor = .bgPrimary
+        tableView.estimatedRowHeight = BasicTableView.estimatedRowHeight
+        tableView.separatorColor = .tableViewSeparator
+        tableView.separatorInset = .leadingInset(frame.width)
+        tableView.estimatedRowHeight = BasicTableView.estimatedRowHeight
```

Comment:
Didn't see this one last time, but it should also be removed. Duplicate of line 36